packages/api/src/utility.ts

```diff
@@ -93,6 +93,11 @@ export function mapSubscriptionsAsCsv(subscriptions: Subscription[]) {
   return csvStr
 }
 
+// Removes commas and new lines within csv content. Avoids disrupted csv formatting.
+function sanitizeCsvContent(input: string | undefined): string {
```

Comment:
Shouldn't we escape how it is described in the RFC:
https://www.ietf.org/rfc/rfc4180.txt
![image](https://user-images.githubusercontent.com/41843343/201965218-8fd48cc3-d9cc-4861-ac70-593af7a86a2e.png)

![image](https://user-images.githubusercontent.com/41843343/201964916-f4dcb8f5-4fc0-44aa-a021-273529bd97b4.png)